src/main/java/frc/robot/subsystems/shooter/ShooterIO.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.shooter;
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.littletonrobotics.junction.AutoLog;
+/** Add your docs here. */
+public interface ShooterIO {
+    @AutoLog
+     public static class ShooterIOInputs {
+     public double drivePositionMeters = 0.0;
+     public double driveVelocityMetersPerSec = 0.0;
```

Comment:
why are there inputs from swerve here? this should just have pivot and flywheel inputs